lib/filtersRegistry.js

```diff
@@ -11,10 +11,52 @@ const { isAsyncFunction } = require('./utils');
  * @param {String} filtersDir Directory where local filters are located.
  */
 module.exports.registerFilters = async (nunjucks, templateConfig, templateDir, filtersDir) => {
+  const hasTypescriptFiles = await checkForTypescriptFiles(templateDir, filtersDir);
+  if(hasTypescriptFiles) {
+    utils.registerTypescript(true)
```

Comment:
In my IDE, the utils seems to be undefined. You probably want to add `const { utils } = require('./utils');`

Reply:
yup, @utnim2  please fix line 4 that for now requires only `isAsyncFunction`